src/GovUk.Education.ExploreEducationStatistics.Data.Api/Mappings/MappingProfiles.cs

```diff
@@ -17,12 +17,12 @@ public MappingProfiles()
             // Null collections will be mapped to null collections instead of empty collections.
             AllowNullCollections = true;
 
-            ConfigureForPermalinks();
+            ConfigureForLegacyPermalinks();
         }
 
-        private void ConfigureForPermalinks()
+        private void ConfigureForLegacyPermalinks()
         {
-            CreateMap<Permalink, PermalinkViewModel>();
+            CreateMap<LegacyPermalink, PermalinkViewModel>();
```

Comment:
Worth going wild and renaming `PermalinkViewModel` to `LegacyPermalinkViewModel`?  Or will this be gone soon enough (or the name reverted) so it's not worth doing?

Reply:
Cool ta